tests/integration/targets/ipam_host/tasks/main.yml

```diff
@@ -0,0 +1,172 @@
+      register: host
+    - name: Get information about the host
+      infoblox.bloxone.ipam_host_info:
+        filters:
+          name: "{{ name }}"
+      register: host_info
+    - assert:
+        that:
+          - host is not failed
+          - host_info.objects | length == 1
+          - host_info.objects[0].id == host.id
+
+    - name: Create a Host (idempotent)
+      infoblox.bloxone.ipam_host:
+        name: "{{ name }}"
+        state: "present"
+      register: host
+    - assert:
+        that:
+          - host is not changed
+          - host is not failed
+
+    - name: Delete a Host (check mode)
+      infoblox.bloxone.ipam_host:
+        name: "{{ name }}"
+        state: "absent"
+      check_mode: true
+      register: host
+    - name: Get information about the Host
+      infoblox.bloxone.ipam_host_info:
+        filters:
+          name: "{{ name }}"
+      register: host_info
+    - assert:
+        that:
+          - host is changed
+          - host is not failed
+          - host_info.objects | length == 1
+
+    - name: Delete a host
+      infoblox.bloxone.ipam_host:
+        name: "{{ name }}"
+        state: "absent"
+      register: host
+    - name: Get information about the  host
+      infoblox.bloxone.ipam_host_info:
+        filters:
+          name: "{{ name }}"
+      register: host_info
+    - assert:
+        that:
+          - host is changed
+          - host is not failed
+          - host_info.objects | length == 0
+
+    - name: Delete a host (idempotent)
+      infoblox.bloxone.ipam_host:
+        name: "{{ name }}"
+        state: "absent"
+      register: host
+    - assert:
+        that:
+          - host is not changed
+          - host is not failed
+
+    - name: Create a Host with comment
+      infoblox.bloxone.ipam_host:
+        name: "{{ name }}"
+        comment: "test comment"
+        state: "present"
+      register: host
+    - name: Get information about the Host
+      infoblox.bloxone.ipam_host_info:
+        filters:
+          name: "{{ name }}"
+      register: host_info
+    - assert:
+        that:
+          - host is not failed
+          - host_info.objects | length == 1
+          - host_info.objects[0].id == host.id
+          - host_info.objects[0].comment == "test comment"
+
+    - name: Create a Host with tags
+      infoblox.bloxone.ipam_host:
+          name: "{{ name }}"
+          tags:
+            tag1: "value1"
+            tag2: "value2"
+          state: "present"
+      register: host
+
+    - name: Get information about the Host
+      infoblox.bloxone.ipam_host_info:
+        filters:
+          name: "{{ name }}"
+      register: host_info
+    - assert:
+        that:
+          - host is not failed
+          - host_info.objects | length == 1
+          - host_info.objects[0].id == host.id
+          - host_info.objects[0].tags.tag1 == "value1"
+          - host_info.objects[0].tags.tag2 == "value2"
+
+    - name: Create an IP space
```

Comment:
Use `setup_ip_space` and `setup_subnet` instead of creating them here .

Reply:
Done